src/query/settings/src/lib.rs

```diff
@@ -111,6 +111,9 @@ impl Settings {
         if conf.query.num_cpus != 0 {
             num_physical_cpus = conf.query.num_cpus;
         }
+        if num_physical_cpus == 0 {
```

Comment:
Printing a warn log here would be better if the num_physical_cpus is 0. It means we get it failed.

Reply:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/172204/208860870-d09ebe90-8808-412b-bbcd-62ff3ece06ee.png">

We can log the settings that reload from the global meta in this PR.

Reply:
I used to log that, but it'll generate many logs in each new session from @Xuanwo advices.